common/src/chain/transaction/mod.rs

```diff
@@ -143,6 +143,30 @@ impl Transaction {
     }
 }
 
+pub struct Transactions {
+    transactions: Vec<SignedTransaction>,
+}
+
+impl Transactions {
+    pub fn new(transactions: &[SignedTransaction]) -> Transactions {
+        Transactions {
+            transactions: transactions.to_vec(),
+        }
+    }
+
+    pub fn transactions(&self) -> &[SignedTransaction] {
+        &self.transactions
+    }
+}
+
+impl From<&[SignedTransaction]> for Transactions {
```

Comment:
There's very little value in this `From` impl... isn't it easier to just call `to_vec()` anyway?